### ISSUE TITLE
Error Log Bug: Incorrect File Coordinates due to Line-Continuation '\'

### ISSUE THREAD

**Issue by tajmone** [pml-lang/converter#34]
I was doing some preparation work to enhance Sublime PML build system so that it would parse error logs and extract info to enable to navigate through the various errors in their source files (see: tajmone/Sublime-PML#16).

While doing some tests, I've encountered a problem with `pmlc` error reports pertaining incorrect positional information being reported for errors that are located after a line-continuation `\`.

The following snippet from the beginning of a PML source:

```pml
[doc title = Malformed Document (standalone) \
    authors = tajmone \
    date = YYYY-MM-DD
```

is reported as:

```
Error: 'YYYY-MM-DD' is an invalid value for parameter 'date'. Reason:
'YYYY-MM-DD' is not a valid ISO date.

[ >>> doc title = Malformed Document (standalone) \
line 1, column 2
X:\<obscured path>\standalone.pml
```

the positional info `line 1, column 2` is incorrect — should be `line 3, column 12`.

To be on the safe side, I've tested this first using Tab indentation (one Tab) and then converting each Tab to 4 spaces, but the result didn't change.

It seems that the problem here is the line continuation `\`, which is not being accounted for when reporting the error position, because other tests that don't use line-continuation were reporting error positions correctly.

**Comment by pml-lang** [comment]
> the positional info line 1, column 2 is incorrect — should be line 3, column 12.

I agree.

> It seems that the problem here is the line continuation \

I tested without line continuation:
```
[doc title = Malformed Document (standalone) authors = tajmone date = YYYY-MM-DD
```
.. and the error message is wrong too:
```
[ >>> doc title = Malformed Document (standalone) authors = tajmone date = YYYY-MM-DD
line 1, column 2
```
Needs to be fixed in the new parser.

**Comment by pml-lang** [comment]
This bug has been fixed in version 2.0.0
